Tw 789 Update Running Flows in Cloud and Block list (#4442)
postmanlabs/postman-docs: src/pages/docs/postman-flows/running-flows-on-the-cloud/webhooks.md

---
title: "Using Flows webhooks"
updated: 2022-11-30
---

> To create a Flow that executes on the Postman servers, create a start block.

Reviewer: ```suggestion
To create a flow that executes on the Postman servers, create a start block.
```

> To create a Flow that executes on the Postman servers, create a start block.

Reviewer 2: ```suggestion
To create a Flow that executes on the Postman servers, create a **Start** block.
```

![Start Block](https://assets.postman.com/postman-labs-docs/cloud-execution/running-flows-on-the-cloud-start.gif)

## Contents

* [Testing Flows locally](#testing-flows-locally)
* [Create a release](#create-a-release)
* [Trigger the webhook](#trigger-the-webhook)

## Testing Flows locally

> Before starting, enter your test data and run it to see the output on your local console. Test data can be added by clicking the gear icon on the start block. The test data can either be in text or JSON format.

Reviewer 2: ```suggestion
Before starting, enter your test data and run it to see the output on your local console. Test data can be added by clicking the gear icon on the **Start** block. The test data can either be in text or JSON format.
```

![Testing Locally](https://assets.postman.com/postman-labs-docs/cloud-execution/running-flows-on-the-cloud-test-data.gif)

## Create a deploy

> Select **Deploy** and your Flow will now be running in the cloud. Selecting the **Runs** section of the right-hand panel will display the deployed Flow was last updated. The webhook url can be seen here as well as in the top left corner of the flow.

Reviewer: ```suggestion
Select **Deploy** and your flow will now be running in the cloud. Selecting the **Runs** section of the right-hand panel will display the deployed Flow was last updated. The webhook URL can be seen here as well as in the top left corner of the flow.
```

> **Important**
>
> Every time you release, it saves a snapshot of your Collections and Environments. If you make any changes to these, you will need to release again.

![Release to cloud](https://assets.postman.com/postman-labs-docs/cloud-execution/running-flows-in-the-cloud-create-deploy.gif)

## Trigger the webhook

> In the runs section, clicking **view** will allow the Flow running in the cloud to be observed and the logs will be visible in your client. Make a POST request to trigger the webhook endpoint and observe your Flow executing in the cloud.

Reviewer: ```suggestion
In the Runs section, selec **View** to observe the flow running in the cloud. The logs will be visible in your client. Make a POST request to trigger the webhook endpoint and observe your flow executing in the cloud.
```

Author: If this can be edited, it looks like there's a typo selec, then I can commit it
